src/util.h

```diff
@@ -110,6 +110,16 @@ void DumpBacktrace(FILE* fp);
 
 #define ABORT() node::Abort()
 
+#define ERROR_AND_ABORT(expr)                                                 \
+  do {                                                                        \
+    /* Make sure that this struct does not end up in inline code, but      */ \
+    /* rather in a read-only data section when modifying this code.        */ \
+    const node::AssertionInfo args = {                                        \
+      __FILE__ ":" STRINGIFY(__LINE__), #expr, PRETTY_FUNCTION_NAME           \
+    };                                                                        \
```

Comment:
This is a performance regression because it creates the struct on every visit. The comment is also wrong because the struct is allocated on the stack, not in RO memory.

Reply:
Thanks, I understand that (but was assuming the compiler would optimize it away as all are compile time. But, assuming anything about the compiler is not good, so I'm wrong).
@bnoordhuis , I'll appreciate a suggestion for fixing this, so that it also works with the gcc4.9 workaround in place.

Reply:
> I'll appreciate a suggestion for fixing this, so that it also works with the gcc4.9 workaround in place.

`static const` like it was in L131

Reply:
Thanks @refack. As I mentioned in one of my above messages, `static const` does not work, because `UNREACHABLE` is called from [here](https://github.com/nodejs/node/blob/ebcd50298e4daf59991a73557445155d8a64dbb1/src/node_file.h#L165), which turns into `inline` function because of the gcc workaround [[here]](https://github.com/nodejs/node/blob/ebcd50298e4daf59991a73557445155d8a64dbb1/src/node_file.h#L152). I got an error, `static` cannot be used inside `inline` function.

Reply:
But maybe it is okay to not call `UNREACHABLE` there because we already have a static assert which will trigger during compilation?

Reply:
😄 https://github.com/nodejs/node/blame/ebcd50298e4daf59991a73557445155d8a64dbb1/src/node_file.h#L165 is my fault.

I believe it's quite Ok to remove the `UNREACHABLE` from there.

Reply:
Thanks! Updated. Let's see if this works now, it worked locally.